docs/development/design/new-sphinx-guides.rst

```diff
@@ -0,0 +1,186 @@
+Proposed contents for new Sphinx guides
+=======================================
+
+This classification follows the `Diátaxis Framework <https://diataxis.fr/>`__.
```

Comment:
I feel like we basically just want a tutorial here. I don't think we need to think about this project as having 4 sections, but more about building a tutorial for Sphinx & RTD. This would be fulfilling the "Tutorial" value for these projects. 

Are we thinking about expanding this work into a refactor of the entire RTD or Sphinx docs? That's where I could see the framework coming into play more.

Reply:
It helped me conceive this as a separate site with its own structure, but after reading

> Are we thinking about expanding this work into a refactor of the entire RTD or Sphinx docs? That's where I could see the framework coming into play more.

I realized that it might be too ambitious. The simplest path would be then:

- Upstream the tutorial to Sphinx
- Write a tutorial on Read the Docs (which I hadn't start working on yet)
- Expand our guides (see my comment above), with the option of upstreaming them.

And talking about upstreaming, I did not reach out yet to Sphinx developers about this. Will do when we are more settled.